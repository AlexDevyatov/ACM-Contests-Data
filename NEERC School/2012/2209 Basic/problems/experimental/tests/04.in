10 5
1 4
5 2
2 5
2 5
1 5
2 5
1 2
3 4
3 5
5 1
3 3 1 0 3
