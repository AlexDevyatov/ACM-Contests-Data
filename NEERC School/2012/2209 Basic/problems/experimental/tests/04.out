1 5 5 2 1 2 2 3 5 1
